### DISCUSSION TITLE
The issue about set dungeon 2, quest chain, and attunement to AQ by Zalausthrafon

### DISCUSSION THREAD

**Discussion by Yehonal** [chromiecraft/chromiecraft#2030] [General]
[The_issue_about_set_dungeon_2_quest_chain_and_attunement_to_AQ__by_Zalausthrafon.pdf](https://github.com/chromiecraft/chromiecraft/files/7315318/The_issue_about_set_dungeon_2_quest_chain_and_attunement_to_AQ__by_Zalausthrafon.pdf)

This PDF contains some suggestions sent by one of our time-traveller friends Zalausthra. We've been working on some of these suggestions.
I upload this here to store it in a publicly accessible place in order to allow you to contribute to this topic.

**Comment by 55Honey** [comment]
What we did so far, is to remove lots of Silithus quests. The issue and PR are in the 59a section of the progression repo.

While this doesn't work towards enabling T0.5, it gates it. I'm also strongly suggesting to temporarily remove the templar set drops. So we also Gate uber powerful catchup items Like earthstrike, which were added with the AQ patch.

**Reply by Zalausthra** [reply]
Hi. I would like to thank Yehonal for posting and support my idea, together to Honey. I can only agree with the latter. I would add that one of these npc's appears in the file I wrote, essential to the summoning of Abyssal Council members at wind stones and greater wind stones. 
Still on the subject of T0.5 sets and related removals, I also remember the deactivation of the npc Geologist Larksbane, useful for the direct evocation of the Duke of Cynders and obtaining, at the end of the quests chain 'The Twilight Mystery", recipes to forging / sewing items with a 63 item level, perfectly in accordance with the current bracket. 
(https://wowgaming.altervista.org/aowow/?quest=8284) 
From a role-playing point of view, to explain this temporal anticipation, one could think of Chromie's need to investigate this precise chain of events and annexes (see T0.5), in which clues to the obscure cause sought may be hidden. Could it be that the Twilight Hammer is running this apocalyptic theater? Or is it perhaps the members of the Infinite Dragonflight? Or, who knows, another unknown entity? As it happened so far, it will be up to the adventurers to search for the truth, repeating the same dungeons several times in order to grasp the coveted details.
You may wonder how the dungeons are connected to this. Well, they are through the mission "Three Kings of Flame", one of the steps of the quests chain for obtaining T0.5. This is because it requires the defeat of Pyroguard Emberseer and Lord Incendius, two bosses respectively present in the dungeons of Upper Blackrock Spire and Blackrock Depths, as well as the Duke of Cynders - summoned through the wind stones of Silithus. This could be a good way to introduce players to the release of Phase B related to the current bracket.
I take this opportunity to anticipate to post, between today and tomorrow, also an idea about attunements in general. Because one cannot speak only of attunements for Ahn'Qiraj. From my point of view, it must be done for every instance in which this is possible, to avoid falling into trivialities.

**Reply by Sealfate** [reply]
Thanks for posting this Yehonal! You did it better than i would do certainly!

Dungeon 0.5 set is very unique set, rare and forgotten. I believe many people are looking forward to obtain this set (including myself) considering a progressive nature of the server. By not adjusting some minor changes (namely Silithus part) you can obtain 6 out of 8 pieces which is enough to activate proc part of the set for all classes. Last item set bonus for all classes is +200 armor which is more or less irrelevant (good addition in PvP mode though).

Last two pieces however are the biggest upgrades because historically Head, Chest and also Legs in WoW had the biggest stats out of all armor slots.

Should we be allowed to obtain full set before we make first step into raid? I think we should. My answer is YES! Let me tell you why.

If you consider when it was released on official classic server (Phase 5- AQ20 & 40) by this time most raiding players had T1 and T2 on farm. Also PvP armor. During this time whole 0.5 has become irrelevant considering the stats, set bonus, cost of whole chain and hussle.

Adding possibility of obtaining this set before first raid wont break a game definitely and it would allow some specs to have an option besides off tier pieces in first two raids. Actual proper use of this set. Many people including myself have no experienced playing with this armor and Chromiecraft is perfect place to make this happen. 

Not only playing with set but seeing this famous long chain quest. It is perfect time to make this happen because people will be farming and running dungeons for months.

I also believe not everyone will commit to obtaining this. Compared to list of pre raid BiS this upgraded set is average but an option nevertheless. It is easily replaceable by dungeons none set pieces, Tier1 and definitely without doubt by Tier 2. 

Let people have fun and enjoy. Please make our wish come true!

**Reply by Zalausthra** [reply]
> Thanks for posting this Yehonal! You did it better than i would do certainly!
> 
> Dungeon 0.5 set is very unique set, rare and forgotten. I believe many people are looking forward to obtain this set (including myself) considering a progressive nature of the server. By not adjusting some minor changes (namely Silithus part) you can obtain 6 out of 8 pieces which is enough to activate proc part of the set for all classes. Last item set bonus for all classes is +200 armor which is more or less irrelevant (good addition in PvP mode though).
> 
> Last two pieces however are the biggest upgrades because historically Head, Chest and also Legs in WoW had the biggest stats out of all armor slots.
> 
> Should we be allowed to obtain full set before we make first step into raid? I think we should. My answer is YES! Let me tell you why.
> 
> If you consider when it was released on official classic server (Phase 5- AQ20 & 40) by this time most raiding players had T1 and T2 on farm. Also PvP armor. During this time whole 0.5 has become irrelevant considering the stats, set bonus, cost of whole chain and hussle.
> 
> Adding possibility of obtaining this set before first raid wont break a game definitely and it would allow some specs to have an option besides off tier pieces in first two raids. Actual proper use of this set. Many people including myself have no experienced playing with this armor and Chromiecraft is perfect place to make this happen.
> 
> Not only playing with set but seeing this famous long chain quest. It is perfect time to make this happen because people will be farming and running dungeons for months.
> 
> I also believe not everyone will commit to obtaining this. Compared to list of pre raid BiS this upgraded set is average but an option nevertheless. It is easily replaceable by dungeons none set pieces, Tier1 and definitely without doubt by Tier 2.
> 
> Let people have fun and enjoy. Please make our wish come true!

I absolutely agree with you. In addition, another element of interest, and not a small one, that I mentioned in my pdf file, is the ability to summon unique bosses during and at the end of mission progression, some of which provide very important items for as many classes.

**Reply by Sealfate** [reply]
It is harmless to make this happen. The only concerning thing is increase level cap which was not a plan for next phase.

Is it big of an issue to increase level cap just because of our suggestion? It is hard to say.

Currently i have pre raid BiS but i have decided to go all in on this set despite losing moderate amount of spell power.

It is not really about losing damage but to own whole set and play with it while it is relevant. It is about experiencing something i never had a chance to do.

**Reply by Zalausthra** [reply]
> It is harmless to make this happen. The only concerning thing is increase level cap which was not a plan for next phase.
[...]

I guess I didn't explain myself well. The goal I was aiming for was to embrace more of the content offered by the game and address it at this same stage to ensure a singular experience for players, myself included. Like you, I've never had the chance to experience this set in the due era and would love to!
